formhero/forms/tests.py

```diff
@@ -1,4 +1,32 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from formhero.apps.models import App
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from formhero.forms.views import CreateForm
+from rest_framework import status
 
 
-# Create your tests here.
+User = get_user_model()
+
+
+class RetrievePostFormTests(APITestCase):
+
+    def test_post_a_new_form(self):
+        app1 = App.objects.create(host_list=["testserver"])
+        handler = 'db'
+        data = {"app": app1.pk, "name": "Szymon", "handler": handler, "config": "test"}
+        url = reverse(CreateForm.URL_NAME, kwargs={'pk': app1.pk})
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(response.data['app'], app1.pk)
+        self.assertEqual(response.data['name'], "Szymon")
+        self.assertEqual(response.data['handler'], handler)
+        self.assertEqual(response.data['config'], "test")
+
+    def test_should_not_allow_to_post(self):
```

Comment:
@szymonzmyslony Similarly to the above, test should be called `test_should_not_allow_to_create_new_form_when_invalid_data_is_posted`